src/app/page/TypeAheadDemo/TypeAheadDemo.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { useState } from "react";
```

Comment:
Is this a temp component?

Reply:
Yes, this is just so you can see the userSearch Component in action, this can be thrown away when you guys actually use the userSearch Component somewhere

Reply:
Thanks for sharing this demonstration! However, I don't think we should add test components into the source code like this.

I think sharing some sample code is enough to demonstrate usage, or taking a moment to see if a storybook component can be set up.

Reply:
Understood, will throw it away then and share a sample code

Reply:
@jwu910 where do I share the sample usage code for this ?

Reply:
Hey @AquiburRKhan You always share as a comment or in this PR description.

You can also consider looking at helping build out storybook components so other devs can see samples. The Storybook library is pretty small right now, and i'm sure additions would be appreciated.